<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>570</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_un">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>USERNAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PASSWORD:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_bg" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>411</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_min">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_close">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎来到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>流币直播间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>做中国最牛的直播间</string>
   </property>
  </widget>
  <zorder>widget_bg</zorder>
  <zorder>btn_login</zorder>
  <zorder>btn_register</zorder>
  <zorder>user_password</zorder>
  <zorder>user_name</zorder>
  <zorder>label_un</zorder>
  <zorder>label_pw</zorder>
  <zorder>toolButton_min</zorder>
  <zorder>toolButton_close</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
